content/en/docs/concepts/services-networking/service.md

```diff
@@ -79,9 +79,13 @@ deploying and evolving your `Services`.  For example, you can change the port
 number that pods expose in the next version of your backend software, without
 breaking clients.
 
-Kubernetes `Services` support `TCP` and `UDP` for protocols.  The default
+Kubernetes `Services` support `TCP`, `UDP` and `SCTP` for protocols.  The default
 is `TCP`.
 
+{{< note >}}
+**Note:** SCTP support is an alpha feature
```

Comment:
Do we want a version number here?

Reply:
Sure. Are you OK with something like "SCTP support is an alpha feature since Kubernetes 1.12"?
Unfortunately, I could not find a good way to include the version as a shortcode, without the usual "FEATURE STATE:..." prefix, which would look strange here in this note, I think.